internal/gitprovider/azure/azure.go

```diff
@@ -94,6 +95,12 @@ func (p *provider) CreatePullRequest(
 		return nil, fmt.Errorf("error getting repository %q: %w", p.repo, err)
 	}
 	repoID := ptr.To(repository.Id.String())
+	labels := make([]adocore.WebApiTagDefinition, 0, len(opts.Labels))
+	for _, label := range opts.Labels {
+		labels = append(labels, adocore.WebApiTagDefinition{
+			Name: &label,
+		})
+	}
```

Comment:
Should work based on [this](https://github.com/Azure/azure-devops-cli-extension/pull/1358/files#diff-003bc1ec1705ca0b976c2786c1ecf1e3370683b0dac7ce8d759dde906d71e073R164)